package.json

```diff
@@ -164,14 +165,19 @@
         "dom-closest": "^0.2.0",
         "dom-scroll-into-view": "^1.2.1",
         "enquire.js": "^2.1.6",
-        "json2mq": "^0.2.0",
         "is-negative-zero": "^2.0.0",
+        "json2mq": "^0.2.0",
```

Comment:
What role does it play?

Reply:
Looks like it just moved down a space - you can see it on line 167 in the original version <-